onedal/cluster/kmeans.py

```diff
@@ -240,7 +282,8 @@ def _fit_backend(self, X_table, centroids_table, module, policy, dtype=np.float3
 
     def _fit(self, X, module, queue=None):
         policy = self._get_policy(queue, X)
-        _, X_table, dtype = self._get_params_and_input(X, policy)
+        is_csr = _is_csr(X)
+        _, X_table, dtype = self._get_params_and_input(X, is_csr, policy)
```

Comment:
`_get_params_and_input` is called only here and part of the results are note used, maybe make sense update it?